kolibri/content/management/commands/importchannel.py

```diff
@@ -65,12 +65,13 @@ def _transfer(self, method, channel_id, path=None):
 
         with filetransfer:
 
-            with self.start_progress(total=filetransfer.total_size) as progress_update:
+            with self.start_progress(total=filetransfer.total_size*1.1) as progress_update:
 
                 for chunk in filetransfer:
                     progress_update(len(chunk), progress_extra_data)
 
-        annotation.update_channel_metadata_cache()
+                channel_import.import_channel_from_local_db(channel_id)
+                progress_update(filetransfer.total_size*0.1, progress_extra_data)
```

Comment:
Both - it was an entirely arbitrary addition, so that the importing took *some* fraction of overall progress.